src/foam/nanos/auth/AuthService.js

```diff
@@ -109,6 +109,27 @@ foam.INTERFACE({
         }
       ]
     },
+    {
+      name: 'checkUserPermission',
```

Comment:
The regular checkPermission() method takes a Permission, which is why it's called checkPermission. The version that takes a String is just called check(). This method just takes a String, so it should be called checkUser(). Or maybe, checkUser() (or maybe userCheck()). Then, for consistency, you might want a second version called userCheckPermission() which takes a Permission. Or name them checkForUser() and checkPermissionForUser().